server/db_mysql.go

```diff
@@ -195,6 +196,39 @@ func (ms *MsqlCache) GetFixity(start string, end string, item string, status str
 	return results
 }
 
+func (ms *MsqlCache) PostFixity(item string) (int, error)  {
```

Comment:
Make name more descriptive, and don't reference the HTTP verb. Maybe `AddFixityForItem` or `ScheduleFixityForItem`.